src/modules/gcloud.rs

```diff
@@ -302,6 +305,37 @@ project = abc
         dir.close()
     }
 
+    #[test]
+    fn project_set_in_env() -> io::Result<()> {
+        let dir = tempfile::tempdir()?;
+        let active_config_path = dir.path().join("active_config");
+        let mut active_config_file = File::create(&active_config_path)?;
+        active_config_file.write_all(b"default")?;
+
+        create_dir(dir.path().join("configurations"))?;
+        let config_default_path = dir.path().join("configurations").join("config_default");
+        let mut config_default_file = File::create(&config_default_path)?;
+        config_default_file.write_all(
+            b"\
+[core]
+project = abc
+",
+        )?;
+
+        let actual = ModuleRenderer::new("gcloud")
+            .env("CLOUDSDK_CORE_PROJECT", "env_project")
+            .env("CLOUDSDK_CONFIG", dir.path().to_string_lossy())
+            .config(toml::toml! {
+                [gcloud]
+                format = "on [$symbol$project]($style) "
+            })
+            .collect();
+        let expected = Some(format!("on {} ", Color::Blue.bold().paint("☁️ env_project")));
```

Comment:
Looks like I broke this when I tried to fix the merge conflicts, I think this should fix it 
```suggestion
        let expected = Some(format!("on {} ", Color::Blue.bold().paint("☁️  env_project")));
```

Reply:
I'll take a look once I get back to my machine. ⌛

Reply:
Yup, that was it!